docs/02_cleos/02_how-to-guides/how-to-submit-a-transaction.md

```diff
@@ -1,18 +1,35 @@
-## Goal
+## Overview
 
-Push a transaction
+This how-to guide provides instructions on how to submit, or push, a transaction using the `cleos` CLI tool.
 
 ## Before you begin
 
 * Install the currently supported version of `cleos`
 
 * Understand the following:
-  * What is a transaction
-  * How to generate a valid transaction JSON
+  * What is a [transaction](https://developers.eos.io/welcome/latest/glossary/index/#transaction).
+  * How to generate a valid transaction JSON.
+    * Consult [cleos push transaction reference](https://developers.eos.io/manuals/eos/latest/cleos/command-reference/push/push-transaction), and pay attention to option `-d` and `-j`.
+    * Consult [push transaction operation](https://developers.eos.io/manuals/eos/latest/nodeos/plugins/chain_api_plugin/api-reference/index#operation/push_transactions) for chain api plug-in, and pay attention to the payload definition.
```

Comment:
This URL should link to the `push transaction` endpoint (singular), not `push transactions`.

Also `endpoint` is more appropriate than `operation`; can simplify to something like:

* Consult the [push transaction] endpoint in the Chain API reference and understand its payload definition.

Reply:
updated